webapp/src/shared/components/templates/RowsPageTemplate/RowsPageTemplate.css

```diff
@@ -1,7 +1,8 @@
-.dispatch-page {
```

Comment:
If you change the name and the folder of the jsx, maybe you should also change the name and folder of the css?

Reply:
I think the name of the folder is already changed. Maybe you meant the name of the base class to `rows-page-template`?

Reply:
Ah, I didn't notice the name change. No problem then.